dissect/target/plugins/os/unix/history.py

```diff
@@ -95,6 +96,7 @@ def parse_generic_history(self, file, user: UnixUserRecord, shell: str) -> Itera
         """
         next_cmd_ts = None
 
+        i = -1
```

Comment:
I'm typing this on mobile atm but you know what I mean for the rest of these as well 😄.